docker-compose.yml

```diff
@@ -9,6 +9,7 @@ x-volumes:
 x-environment:
   &default-environment
     LAGOON_ROUTE: ${LOCALDEV_URL:-http://govcms-lagoon.docker.amazee.io}
+    X_FRAME_OPTIONS: ${X_FRAME_OPTIONS:-SameOrigin}
```

Comment:
Slightly different to D8: We need this env variable injected in more than just Nginx (see above comment about `settings.php`)